### PR TITLE
Add changelog updater action

### PR DESCRIPTION
**Pull request by sophiaaar**
Add changelog updater workflow and script

Python file tested [here](https://github.com/modl-ai/UnityPlugin/actions/runs/4597660127/jobs/8120673172) for [this PR](https://github.com/modl-ai/UnityPlugin/pull/393)

### CODE REVIEW

File: scripts/changelog.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python
```
Comment (sysnet-ai) [inline scripts/changelog.py:1]: Minor: It took me a tiny bit to fully understand the flow in this file, a few comments would go a long way.

**Review by sysnet-ai — APPROVED** [review]
I think a few comments on the python file would make this a stellar PR.

🦾

File: scripts/changelog.py (hunk 1 above)
Comment (sophiaaar) [reply scripts/changelog.py]: done!

**Review by sysnet-ai — APPROVED** [review]
:shipit:

**Pull request merged**